bindings/go/nvml/nvml_test.go

```diff
@@ -42,6 +42,14 @@ func BenchmarkDeviceCount1(b *testing.B) {
 	Shutdown()
 }
 
+func TestCudaDriverVersion(t *testing.T) {
+	Init()
+	defer Shutdown()
+
+	_, _, err := GetCudaDriverVersion()
+	check(err, t)
+}
```

Comment:
As nvidia-smi do not expose a query option to only get the Cuda version, we can't test more than this